website/docs/docs/getting-started/2-your-first-test.md

```diff
@@ -0,0 +1,41 @@
+---
+id: your-first-test
+title: Writing your first test
+---
+
+To demonstrate how you can write tests, let's install the TodoMVC app from BigTest:
+```
+$ yarn add -D @bigtest/todomvc
+```
+
+Assuming you went with the default settings for `bigtest.json`, the app command should be configured as `yarn start`:
+```json
+{
```

Comment:
It this json the default content from `bigtest.json`?

Reply:
No, I omitted some of the properties of the full default config file. If you run `bigtest init` and hit enter all the way through (so it's only default suggested values), this is what it would look like:
```json
{
  "port": 24000,
  "launch": [
    "chrome.headless"
  ],
  "app": {
    "command": "yarn start",
    "env": {
      "PORT": 3000
    },
    "url": "http://localhost:3000"
  },
  "testFiles": [
    "test/**/*.test.{ts,js}"
  ]
}
```